README.md

```diff
@@ -4,49 +4,230 @@ Provides resources for installing and configuring both PowerDNS authoritative an
 
 ## Requirements
 
-IMPORTANT: Please read the Compatibility Notes version below since there is breaking changes between 2 and 3 versions of this cookbook.
+IMPORTANT: Please read the Deprecations and Compatibility Notes sections below since there is breaking changes between 2 and 3 versions of this cookbook.
+
+### Deprecations
+
+  - The recipe and attribute based way of setting different PowerDNS installs is completely deprecated, there is not  attributes in the newest version of this cookbok and just a simple empty `default` recipe.
+  - `pdnsrecord` and `domainrecord` resources have been deprecated since they were coupled with sqlite3 backend.
+  - Ubuntu 12.02 support has been removed, if you want this platform to be supported PRs are welcome, see Contributing  section at the end of this document.
 
 ### Compatibility Notes
 
-**This cookbook is being completely rewritten, transitioning from an attribute centric design to a newer resource based design. The current 3.0 version the resource only supports recursors, being the authoritative server the next feature, destined for the 3.1 release which will be released soon. If you want to keep your authoritative PowerDNS installs pin your cookbook to the latest 2.5.0 version.**
+**This cookbook is being completely rewritten, transitioning from an attribute centric design to a newer resource based design. 
+
+TLDR: 
+
+BREAKING CHANGES, Please pin your PowerDNS installs pin your cookbook to the latest 2.5.0 version. We also advise to read this document carefully.
+**
+
+The current version of the cookbook provides basic support for recursors and authoritative servers with a handful of platforms, backends and init systems supported. You can find what is supported in this table:
+
+
+| Platform | Backends         | Init Systems |
+|----------|------------------|--------------|
+| Debian   | bind, postgresql | SysVinit     |
+| CentOS   | bind, postgresql | SysVinit     |
 
 ### Platforms:
 
-* Ubuntu (14.04, 16.04)
-* Debian (8)
-* CentOS (6.8, 7.2)
```

Comment:
Again, what's the justification for removing support for newer and only leaving older platforms.